selfdrive/car/hyundai/interface.py

```diff
@@ -63,7 +63,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
     elif candidate == CAR.SONATA_LF:
       ret.lateralTuning.pid.kf = 0.00005
-      ret.mass = 4497. * CV.LB_TO_KG
+      ret.mass = 2040. + STD_CARGO_KG
```

Comment:
Some of these might've been wrong initially actually. Google says the 2019 Sonata has a curb weight of 3,247 to 3,527 lbs. But is the LF a special varianrt?